src/FlightDisplay/FlyViewMap.qml

```diff
@@ -572,4 +573,16 @@ FlightMap {
         }
     }
 
+    MapScale {
```

Comment:
Adding another MapScale control at this level of the ui hierarchy screws up the layering of ui elements for custom build overrides. Why can't this be fixed just in FlyViewWidgetLayer?

Reply:
Because the map is a component itself when you popup in a new window. 
Are you suggesting to not have the scale after you popup ?

Reply:
Hmm, ok right. I didn't realize the widgets are not shown at all in this mode. In fact I had no idea you could even do this! Although duplicating the MapScale makes my skin crawl a bit it's the simplest way to fix this for now.